### ISSUE TITLE
TypeError: Cannot set property 'definition' of undefined

### ISSUE THREAD

**Issue by carlevison** [bmewburn/intelephense#176]
```
server: TypeError: Cannot set property 'definition' of undefined
    at t.DocumentationService.<anonymous> (/usr/local/lib/node_modules/intelephense/lib/intelephense.js:39:125428)
    at Generator.next (<anonymous>)
    at o (/usr/local/lib/node_modules/intelephense/lib/intelephense.js:39:117666)
    at <anonymous>
```

Seeing this with quite a few methods in Laravel from inside Controller.php such as ```validate``` and ```dispatch```as a few examples.

I would also like to thank you for your amazing work.

* OSX 10.14 - Intelephense 1.0.1

**Comment by Braunson** [comment]
Getting this also - Win 10 - Intelephase 1.0.1

**Comment by Naoray** [comment]
Same - Ubuntu 18.10 - intelphense 1.0.1

**Comment by nanoarmando** [comment]
+1 MacOS 10.14.3

**Comment by bmewburn** [comment]
fixed in 1.0.2

**Comment by nanoarmando** [comment]
Version 1.0.2, working pretty good, the new features are awesome.. Congrats @bmewburn
